pkg/comp-functions/functions/vshnmariadb/mariadb_deploy.go

```diff
@@ -473,3 +497,27 @@ func getMariaDBRootPassword(secretName string, svc *runtime.ServiceRuntime) ([]b
 	}
 	return secret.Data["mariadb-root-password"], nil
 }
+
+// we effectively have to allow all traffic that the service can be
+// accessed via the loadbalancer.
+func addLoadbalancerNetpolicy(svc *runtime.ServiceRuntime, comp *vshnv1.VSHNMariaDB) error {
```

Comment:
why this is the case? why the traffic has to be allowed from other instance namespaces?

Reply:
Otherwise, the external loadbalancer connection is also blocked. I'll open a  followup to do research if we can still somehow allow external connections while block internal connections.